docs/hub/datasets-viewer-sql-console.md

```diff
@@ -0,0 +1,24 @@
+# SQL Console: Query datasets in the browser
+
+You can run SQL queries on the dataset in the browser using the SQL Console. The SQL Console is powered by [DuckDB](https://duckdb.org/) WASM and runs entirely in the browser. You can access the SQL Console from the dataset page by clicking on the **SQL Console** badge.
+
+<div class="flex justify-center">
+    <img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/sql_console/sql-console-histogram.png"/>
+    <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/sql_console/sql-console-histogram-dark.png"/>
+</div>
+
+<p class="text-sm text-center italic">
+    To learn more about the SQL Console, see the <a href="https://huggingface.co/blog/sql-console" target="_blank" rel="noopener noreferrer">SQL Console blog post</a>.
+</p>
+
+
+Through the SQL Console, you can:
+
+- Run [DuckDB SQL queries](https://duckdb.org/docs/sql/query_syntax/select) on the dataset (_checkout [SQL Snippets](https://huggingface.co/spaces/cfahlgren1/sql-snippets) for useful queries_) 
+- Share results of the query with others via a link (_check out [this example](https://huggingface.co/datasets/gretelai/synthetic-gsm8k-reflection-405b?sql_console=true&sql=FROM+histogram%28%0A++train%2C%0A++topic%2C%0A++bin_count+%3A%3D+10%0A%29)_) 
+- Download the results of the query to a parquet file 
+- Embed the results of the query in your own webpage using an iframe 
+
```

Comment:
Maybe add a section with examples ? like a code snippet + like per example

some ideas: 

1. filter data (by value, and/or with string matching)
1. group by + count labels
1. concatenate and format strings from multiple columns
1. extract strings with regex

This is also the kind of content we can reuse for the duckdb cli/python client docs on HF at https://huggingface.co/docs/hub/main/datasets-duckdb

Reply:
good point yeah I left it smaller since it was part of the original dataset viewer docs, but not that it has it's own page probably can use the content and examples 👍

Reply:
let me know what you think about the new examples